### PR TITLE
Use refactored PyCIEMSS lib

### PR DESCRIPTION
**Pull request by fivegrant**
This PR refactors the service to use PyCIEMSS v0.1. The changes to the API are kept to a minimum but alterations are unavoidable since we are moving from "big box" to "small box" (atomic) operations.

## PyCIEMSS features that still need to be incorporated into the service
- [x] Pass `parameters.dill` into `simulate` (i.e. using `inferred_parameters` from `calibrate` for `sample`)
- [x] Reimplement interventions (unclear which kind of interventions I should expose)
- [ ] Fix solution mappings for `ensemble-simulate` to work with the new format

### Regressions that have now been fixed but need to be incorporated into service
- [x] Progress hook with additional 'loss' field.

## Regressions in PyCIEMSS
- [ ] Visualizations
- [x] Progress hook
- [ ] Ensemble Calibrate
- [ ] Optimize operations
- [ ] ~~[Logging decorator missing from `ensemble-sample` (very minor)](https://github.com/ciemss/pyciemss/issues/440)~~

### CODE REVIEW

**Comment by fivegrant** [comment]
Please note the following regressions
- #52
- #53
- #54
- #55
- #56

tl;dr: `simulate` and `calibrate` have been successfully refactored while there are various issues with everything else

## Dashboard Results
image
`simulate` and `calibrate` now work for all except one models. This includes stockflow and regnets. All of the other other operations no longer pass

**Review by liunelson — APPROVED** [review]
